lib/spack/spack/ci.py

```diff
@@ -499,6 +499,7 @@ def generate_gitlab_ci_yaml(env, print_summary, output_file,
             'pushd ./spack && git checkout "{0}" && popd'.format(
                 custom_spack_ref),
             '. "./spack/share/spack/setup-env.sh"',
+            'spack env activate .',
```

Comment:
@scottwittenburg this looks like it only activates the environment if it's using a custom spack repo? Don't we need this to be regardless of any options?

Reply:
Yes indeed, good catch, thank you!